jumpscale/sals/zos/gateway.py

```diff
@@ -37,7 +54,7 @@ def sub_domain(self, gateway_id: str, domain: str, ips: List[str], pool_id: int)
         for ip in ips:
             if not _is_valid_ip(ip):
                 raise Input(f"{ip} is not valid IP address")
-
+        self.correct_domain(domain)
```

Comment:
hmm what does this call do? as far as i understood it returns a corrected domain, but where is that captured?